### PR TITLE
Pep8 and some unit tests

### PR DESCRIPTION
**Pull request by isoutham**
Fixes up a bunch of pep8 failure (trying to get rid of some jenkins failures)
Also the first steps to a unit testing framework for the python code in the systemvm

### CODE REVIEW

**Comment by wilderrodrigues** [comment]
It seems this PR came with the same commit as the previous one, that's why Travis is now building the #262 again.

Did you create a second PR with the same commit + extra fixes? Should we just ignore the previous one?

**Comment by isoutham** [comment]
It looks like I have somehow managed to put the second PR in the first (was
a seperate branch) not quite sure what I did here.

Ignore the first and treat it as one I would say but if you feel otherwise,
let me know.

File: systemvm/patches/debian/config/opt/cloud/bin/configure.py [hunk 1]
```diff
@@ -555,35 +558,47 @@ def forward_vr(self, rule):
...
-              ( rule['public_ip'],
+              (
+                rule['public_ip'],
                 self.getDeviceByIp(rule['public_ip']),
                 rule['protocol'],
                 rule['protocol'],
                 self.portsToString(rule['public_ports'], ':'),
-              )  
+              )
+        fw7 = "-A FORWARD -i %s -o %s -p %s -m %s --dport %s -m state --state NEW -j ACCEPT" % \
```
Comment (bhaisaab) [inline systemvm/patches/debian/config/opt/cloud/bin/configure.py]: @isoutham this PR says pep8/unit test fixes, why are we adding a forwarding rule here?

**Comment by wilderrodrigues** [comment]
@bhaisaab 

For some reason this PR got the same commit of the previous PR (#262). @isoutham told me to ignore the previous one.

So I'm now testing this and it LGTM so far. I will just test redundant VPC and merge it. Below few details of the tests I have done manually:

* Create VPC
* Create Tier
* Create ACL + 2 rules
* Associate ACL with Tier
* Create VM
* Acquire new IP + Add PF rule

Then ping + ssh

[wrodrigues@mct-wrodrigues cloudstack]$ ping 192.168.23.8
PING 192.168.23.8 (192.168.23.8) 56(84) bytes of data.
64 bytes from 192.168.23.8: icmp_seq=1 ttl=64 time=3.24 ms
64 bytes from 192.168.23.8: icmp_seq=2 ttl=64 time=3.76 ms
64 bytes from 192.168.23.8: icmp_seq=3 ttl=64 time=3.96 ms
64 bytes from 192.168.23.8: icmp_seq=4 ttl=64 time=4.08 ms
^C
--- 192.168.23.8 ping statistics ---
4 packets transmitted, 4 received, 0% packet loss, time 3004ms
rtt min/avg/max/mdev = 3.241/3.762/4.080/0.330 ms
[wrodrigues@mct-wrodrigues cloudstack]$ ssh root@192.168.23.8
The authenticity of host '192.168.23.8 (192.168.23.8)' can't be established.
ECDSA key fingerprint is 3f:64:cc:0f:72:09:9a:e4:dd:c3:1b:a0:0c:17:f7:db.
Are you sure you want to continue connecting (yes/no)? yes
Warning: Permanently added '192.168.23.8' (ECDSA) to the list of known hosts.
root@192.168.23.8's password:

**Pull request merged**